spec/controllers/employments_controller_spec.rb

```diff
@@ -110,16 +99,38 @@ def delete_gsi
   end
 
   describe 'update' do
-    describe 'if email is changed' do
-      let(:email) { 'burney@gmail.com' }
+    def update_and_check_if_hired(email, gsi_amount_change)
```

Comment:
Assignment Branch Condition size for update_and_check_if_hired is too high. [26.08/15]